src/test/java/com/zimbra/cs/ldap/ZLdapFilterFactorySimulator.java

```diff
@@ -30,17 +30,17 @@ public ZLdapFilter velodromeAllGroupsByDomain(String s)
 
   public ZLdapFilter allAddressLists()
   {
-    return null;
+    return new FakeLdapFilter(FilterId.ALL_ADDRESS_LISTS);
```

Comment:
Is this change related to colors attribute

Reply:
It is a modification I need to make a test work.